CSV_Trans_Col_To_Rec_.py

```diff
@@ -1,71 +1,71 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# vim:fenc=utf-8
+# vim:fenc=utf-8 
 #! /usr/bin/env python
 # csvファイル読み込みライブラリをImport
 import os
 import csv
 
 def trance(I_files,O_files,begin,column,repeat):
-#+---------+---------+---------                
+#+---------+---------+---------                 
 # I_files :     入力ファイル名（Fullpath)
 # O_files :     出力ファイル名（Fullpath)
 # keys :        行に展開しない列項目数（粒度やキーとしての役割）
 # degin :       開始は0列になるので13列目なら12
 # column :      7列分を移動するなら7
 # repeat :      列→行変換を繰り返す回数
-#+---------+---------+---------                
-#       入力ファイルを文字コードutf8で開く
+#+---------+---------+---------                 
+#       入力ファイルを文字コードutf8で開く 
         f = open(I_files, 'r',encoding="utf-8_sig")
         reader = csv.reader(f)
-#       出力ファイルを文字コードutf8で開く
+#       出力ファイルを文字コードutf8で開く 
         fo = open(O_files, 'w',encoding="utf-8_sig")
         writer = csv.writer(fo, lineterminator='\n')
-#       入力ファイルを読み込み、レコード分繰り返し
+#       入力ファイルを読み込み、レコード分繰り返し 
         for i, row in enumerate(reader):
-#               最初のレコードをHeaderとして処理
+#               最初のレコードをHeaderとして処理 
                 if i == 0:
-                        #Hearder 編集
+                        #Hearder 編集 
                         new = row[0:begin+column]
                         writer.writerow(new)
                         continue                        
-#+---------+---------+---------                
+#+---------+---------+---------                 
 #               以降、データレコードとしての処理を実施
-#+---------+---------+---------
+#+---------+---------+--------- 
 #               beginに列を行へ移動する開始位置を設定
                 start = begin
-#               移動する列項目数を設定
-#               列→行変換を指定回数分実施する。
+#               移動する列項目数を設定 
+#               列→行変換を指定回数分実施する。 
                 for _ in range(repeat):
```

Comment:
If we use `while` here with condition from line#45, we don't need `repeat` parameter anymore (can be deleted from the code):
```suggestion
                while start < len(row):
```